cli/wallet/file/ed25519.go

```diff
@@ -0,0 +1,53 @@
+package file
+
+import (
+	"encoding/base64"
+
+	"github.com/oasisprotocol/curve25519-voi/primitives/ed25519"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
+)
+
+// ed25519rawSigner is an in-memory signer that allows deserialization of raw ed25519 keys for use
+// in imported wallets that don't use ADR 0008.
```

Comment:
what do we use in the relatively raw signers in oasis-core, if not raw ed25519 keys?

Reply:
and if not 'raw ed25519 keys,'

1. why make the CLI use raw ed25519 keys?
2. shall we instead add this functionality directly to oasis-core?

Reply:
> what do we use in the relatively raw signers in oasis-core (file, memory), if not raw ed25519 keys?

We do, but they cannot be used here as they don't support loading the raw private key value (you can only generate a new random key or generate one from a RFC 8032 seed).

Reply:
would you be opposed to adding that import functionality to the memory signer?

Reply:
I wouldn't be, but I would rather avoid the need for this PR to depend on an unreleased version of Oasis Core. Can you raise an issue in Oasis Core (maybe also for exposing the chain context separator) and then once it is part of a stable release we can change it here.